### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
- FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen sami.mikael.hiltunen@cern.ch

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -21,9 +21,6 @@
...
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
-#
-# TODO: Add development versions of some important dependencies here to get a
-#       warning when there are breaking upstream changes, e.g.:
-#
-#     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
-#     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
+
+-e git+git://github.com/inveniosoftware/invenio-ext.git#egg=invenio-ext
```
Comment (jirikuncar) [inline requirements-devel.txt:26]: - missing invenio-base, invenio-upgrader

File: setup.py [hunk 2]
```diff
@@ -37,18 +37,21 @@
     'Flask>=0.10.1',
     'six>=1.7.2',
     'Invenio>=2.0.3',
```
Comment (jirikuncar) [inline setup.py]: remove `Invenio>=2.0.3` and add invenio-base

File: setup.py [hunk 3]
```diff
@@ -37,18 +37,21 @@
...
+    'invenio-ext>=0.1.0',
```
Comment (jirikuncar) [inline setup.py]: missing invenio-upgrader

**Comment by SamiHiltunen** [comment]
If you need this for old release, please ping me.

**Pull request closed without merging**